### PR TITLE
apple-web

### PR DESCRIPTION
**Pull request by EdSacasa**

### CODE REVIEW

**Comment by EdSacasa** [comment]
update

**Comment by EdSacasa** [comment]
fixed linter erros

File: index.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            <img class="w100" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c_2x.jpg" alt="iPhone 5c">
+          </section>
+          <section>
+            <img class="w100" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s_2x.jpg"  alt="iPhone 5s. Forward thinking.">
+          </section>
+          <section>
+            <img class="w100" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_30_years_2x.jpg"  alt="Happy Birthday, Mac. Explore 30 years of innovation on the Mac.">
+          </section>
+          <section>
+            <img class="w100" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility_2x.jpg"  alt="Supplier Responsibility The latest results of our worldwide efforts.">
```
Comment (uche-inyama) [inline index.html]: The **```tags```** should be wrapped with an **``` <a>tag```** with **```href='#'```** . When you [click](https://web.archive.org/web/20140301004610/http://www.apple.com/) on the images, they should take you to another webpage.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      </p>
+    </section>
+    <section class="right">
+      <ul class="bottom-links">
+        <li>
+          <a href="#">Site Map</a>
+        </li>
+        <li>
+          |
+        </li>
```
Comment (uche-inyama) [inline index.html:85]: [**OPTIONAL**]
This is not the best way to implement this design, inspect the webpage on [apple.com](https://web.archive.org/web/20140301004610/http://www.apple.com/), for reference.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi Ed :wave: 

Everything looks almost fine, just few changes to make:

- [ ] For the navigation bar, the color is not uniform, there are two shades of gray in your design. kindly fix that.
- [ ] The search field on the navigation bar is not properly positioned.

              This is how it looks on my screen

Screenshot from 2019-10-24 08-26-08

- [ ] Look below this feedback, for the other comments.

    #### ```After implementing these changes submit another for a code review.```

#### ``` Have a great day``` :slightly_smiling_face: 

@ Uchechukwu Henry Inyama

**Comment by suretrust** [comment]
Hi @EdSacasa,

Great job, **let's get this merged. :+1:** 

**Good luck with your future projects and happy coding!**

**Review by suretrust — APPROVED** [review]

**Pull request merged**